src/app/zap-templates/zcl/data-model/chip/boolean-sensor-configuration-cluster.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0080"/>
+    <field name="Visual" mask="0x1"/>
+    <field name="Audible" mask="0x2"/>
+    <field name="AlarmSupress" mask="0x4"/>
```

Comment:
```suggestion
    <field name="AlarmSuppress" mask="0x4"/>
```

Please carefully go through this XML and compare it to the spec.  The fact that @andy31415 and I keep finding divergences is bad.  The fact that we are having to go through this character by character multiple times is also bad...

Reply:
Well, there are some mistakes but some of it is because I made this based on the draft spec. I will go through it once again.